static/src/javascripts/lib/geolocation.js

```diff
@@ -10,10 +11,17 @@ const editionToGeolocationMap = {
 };
 const daysBeforeGeolocationRefresh = 10;
 
-const getFromStorage = () => storage.local.get(storageKey);
+const getFromStorage = () => getCookie('GU_geo_country') || storage.local.get(storageKey);
```

Comment:
This seems redundant with line 25, where we check `getFromStorage()`, after having looked against the cookie.

Would it be worth simply updating its use in `support-utilities.js` to simply `get()`?

https://github.com/guardian/frontend/blob/5b970cd7308175cfc1bcae2d4fb8c06ee13c5fa0/static/src/javascripts/projects/common/modules/commercial/support-utilities.js#L8-L10

[This seems to be its only use outside this file][1], and it means it could be removed from thee exports.

[1]: https://github.com/search?q=repo%3Aguardian%2Ffrontend+getFromStorage&type=code

Reply:
Indeed, I'm inclined to do that refactor in a separate PR as there's few more things that could be refactored, including using guarrdian/libs `getLocale` which contains a lot of duplicated logic. Also with memoizing the location in the getLocale it means we are improving a lot of these fetches.

Reply:
With the latest changes in libs/getLocale it just makes sense to not duplicate them again here, so will be doing the refactor in this PR. Will keep the PR WIP until we first merge libs one & have a solution about geolocation override from RR.